src/esm_runscripts/prepare.py

```diff
@@ -672,6 +674,50 @@ def set_parent_info(config):
     return config
 
 
+def add_vcs_info(config):
+    """
+    Adds version control system information in a plain text yaml file under the
+    experiment log directory for both the models and the esm-tools.
+
+    Parameters
+    ----------
+    config : dict
+        The experiment configuration
+
+    Returns
+    -------
+    config : dict
```

Comment:
As the dicts are mutable there is no need to return them explicitly.

Reply:
Yes, this is the problem. They were all wrong from the beginning 🤣 . Well, not wrong but let's say misleading.